assets/js/modules/adsense/datastore/settings.test.js

```diff
@@ -253,11 +253,11 @@ describe( 'modules/adsense settings', () => {
 				expect( registry.select( STORE_NAME ).canSubmitChanges() ).toBe( true );
 
 				registry.dispatch( STORE_NAME ).setAccountID( '0' );
-				expect( () => registry.select( STORE_NAME ).__dangerousCanSubmitChanges() )
+				expect( () => registry.select( STORE_NAME ).validateSubmitChanges() )
```

Comment:
See above comment, the previous name should be kept here.

Reply:
Updated.